templates/includes/CartActions.ss

```diff
@@ -0,0 +1,16 @@
+<% if $isAllowedInCart %>
+    <p class="cart-actions">
+    <p>
+        <a class="docCart-remove-link<% if not $isInCart %> hidden<% end_if %>"
+           href="$getActionLink('remove')"><%t DMSDocument.REMOVE_FROM_CART "Remove from cart" %> </a>
+        <a class="docCart-add-link<% if $isInCart %> hidden<% end_if %>"
+           href="$getActionLink('add')"><%t DMSDocument.ADD_TO_CART "Request a printed copy" %> </a>
+        <% if $isInCart %>
+            &bull;
+            <a class="docCart-checkout-link<% if not $isInCart %> hidden<% end_if %>" href="$getActionLink('checkout')"
```

Comment:
Great one. Shouldn't just rely on the reset.css provided by the theme. Added one to the module.